ADO.NET/SampleApp.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('Windows'))">$(TargetFrameworks);net462</TargetFrameworks>
+    <Architecture>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant)</Architecture>
+    <RuntimeIdentifier Condition="$([MSBuild]::IsOSPlatform('Windows'))">win-$(Architecture)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="$([MSBuild]::IsOSPlatform('Linux'))">linux-$(Architecture)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="$([MSBuild]::IsOSPlatform('OSX'))">osx-$(Architecture)</RuntimeIdentifier>
+    <Configuration>Release</Configuration>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Vertica.Data" Version="23.3.0" />

```

Comment:
Did you mean to change the version to rc0?

Reply:
I figured the example should point to the release version, even if it isn't out yet, and the note in the README mentions what to change in order to reference pre-release versions, such as `23.3.0-rc`.

Although we could have the sample point to `23.3.0-rc` by default and we could update it to `23.3.0` when the official release is published.

Reply:
I think the way it is now is fine.